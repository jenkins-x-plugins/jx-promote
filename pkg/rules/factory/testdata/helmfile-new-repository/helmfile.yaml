repositories:
- name: yourorg
  url: https://yourorg.example.com/charts
- name: dev
  url: http://something/else
releases:
- name: dbmigrator
  labels:
    job: dbmigrator
  chart: ./dbmigrator
