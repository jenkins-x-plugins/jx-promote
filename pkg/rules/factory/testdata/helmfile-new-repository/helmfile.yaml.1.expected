filepath: ""
repositories:
- name: yourorg
  url: https://yourorg.example.com/charts
- name: dev
  url: http://something/else
- name: dev2
  url: http://chartmuseum-jx.34.78.195.22.nip.io
releases:
- chart: ./dbmigrator
  name: dbmigrator
  labels:
    job: dbmigrator
- chart: dev2/myapp
  version: 1.2.3
  name: myapp
  namespace: jx
templates: {}
renderedvalues: {}
